<?xml version="1.0" encoding="UTF-8"?>
<project version="4">
  <component name="DataSourceManagerImpl" format="xml" multifile-model="true">
    <data-source source="LOCAL" name="Django default" uuid="e726b11c-f06e-479b-9290-a36770701f16">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/auth_demo/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\HP User\PycharmProjects\auth_demo\db.sqlite3</jdbc-url>
    </data-source>
    <data-source source="LOCAL" name="Django default2" uuid="9551a57d-22ea-42e1-8458-3eefbbb4bd79">
      <driver-ref>sqlite.xerial</driver-ref>
      <synchronize>true</synchronize>
      <imported>true</imported>
      <remarks>$PROJECT_DIR$/auth_demo/settings.py</remarks>
      <jdbc-driver>org.sqlite.JDBC</jdbc-driver>
      <jdbc-url>jdbc:sqlite:C:\Users\HP User\PycharmProjects\auth_demo\db.sqlite3</jdbc-url>
    </data-source>
  </component>
</project>